src/std/net/s3/api.ss

```diff
@@ -0,0 +1,242 @@
+;;; -*- Gerbil -*-
+;;; (C) vyzo
+;;; AWS S3 client
+(import "sigv4"
+        :std/net/request
+        :std/misc/func
+        :std/contract
+        :std/net/uri
+        :std/crypto/digest
+        :std/text/hex
+        :std/xml
+        :std/error
+        :std/sugar
+        :std/srfi/19)
+(export (struct-out s3-client bucket) S3ClientError)
+
+; precomputed empty sha256
+(def emptySHA256 #u8(227 176 196 66 152 252 28 20 154 251 244 200 153 111 185
+                     36 39 174 65 228 100 155 147 76 164 149 153 27 120 82 184 85))
```

Comment:
let's compute this instead of hardcoding it.

Reply:
You can use syntax-eval if you want the computation to happen at compile-time!

Reply:
I forget the power of Lisps haha